userspace/libsinsp/gen_filter.cpp

```diff
@@ -302,9 +308,10 @@ void gen_event_filter_factory::filter_fieldclass_info::wrapstring(const std::str
 	}
 }
 
-std::string gen_event_filter_factory::filter_fieldclass_info::as_markdown(const std::set<std::string>& event_sources)
+std::string gen_event_filter_factory::filter_fieldclass_info::as_markdown(const std::set<std::string>& event_sources, bool include_deprecated)
```

Comment:
I think the usage of `incude_deprecated` is missing in this function :thinking:

Reply:
You are right! 😅